src/main/webapp/app/shared/markdown-editor/markdown-editor.component.html

```diff
@@ -23,7 +23,7 @@
                         <label for="file-upload" class="markdown-editor__file-label d-inline">
                             <div class="row mx-0 align-items-baseline">
                                 <span class="col upload-subtitle">{{ 'artemisApp.markdownEditor.fileUpload' | translate }}</span>
-                                <a class="col-auto" href="https://guides.github.com/features/mastering-markdown/">
+                                <a class="col-auto" href="http://demo.showdownjs.com">
```

Comment:
I think this help link should point to a list of markdown formats. Demoing markdown is also possible in the preview on Artemis itself.

Reply:
But this is the official standard that we support. I specifically checked it. the previous link did not show the exact version but a more generic one. And the demo page includes many examples which are not available as commands in our case. Finding them and trying those out might be helpful for users.

One issue I see is that the link is not very accessible. I bet no one found it until now